### PR TITLE
Update suite.yml for v1.11.1+suite.2 release

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
PR to prep for the January OSS suite release

Outstanding issues/PRs:
- [x] cyberark/conjur-oss-helm-chart#60
- [x] cyberark/conjur-oss-helm-chart#120
- [x] cyberark/conjur-cli#305
- [x] cyberark/ansible-conjur-host-identity#45
- [x] cyberark/ansible-conjur-collection#53 

Release page: https://github.com/cyberark/conjur-oss-suite-release/wiki/v1.11.1-suite.2
Draft release notes: https://gist.github.com/izgeri/2a3e73f6f9b8c3ba9d7aa1233f1fa191

Open questions / issues:
- In the last release we announced we'd be dropping support for OpenShift 3.9 / 3.10 in this release - those changes are not already released in the suite components, so do we still want to announce it here?
   - We will announce this in the next suite release, along with the new support for OCP 4.3 / 4.5
- cyberark/conjur-docs#1015 - issue to update the Ansible docs should be completed with this suite release. Updated content has been appended to the issue.
   - Will be added to the docs issue with the draft release notes.

### What ticket does this PR close?
n/a

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

#### Release PRs
**If you are preparing for a release**, are the following items complete?
- [x] The PR title / description clearly state the suite release version.
- [x] The [Jenkins dashboard](https://jenkins.conjur.net/view/OSS%20Suite%20Components/) shows no ongoing
      build failures for any included components.
- [x] The PR includes a link to a markdown release notes draft - this can be
     generated by running:
     ```
     summon -p keyring.py   --yaml 'GITHUB_TOKEN: !var github/api_token' \
       ./parse-changelogs \
       -v v1.11.1+suite.2 \
       -t release
       -o RELEASE_NOTES.md
     ```
- [ ] The PR includes PM-approved "What's new" text.

### CODE REVIEW

**Comment by izgeri** [comment]
Suggested "What's New" content is available in the gist here: https://gist.github.com/izgeri/2a3e73f6f9b8c3ba9d7aa1233f1fa191#whats-new

File: suite.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@ section:
         url: https://github.com/cyberark/conjur-oss-helm-chart
         description: Helm chart for deploying Conjur OSS.
         upgrade_url: https://github.com/cyberark/conjur-oss-helm-chart/tree/master/conjur-oss#upgrading-modifying-or-migrating-a-conjur-oss-helm-deployment
-        version: v2.0.2
+        version: v2.0.3
```
Comment (alexkalish) [inline suite.yml:19]: Looking at the changelog, this probably should have been v2.1.0, due to the OCP addition.

File: suite.yml [hunk 2]
```diff
@@ -66,13 +66,21 @@ section:
...
+          from within your Ansible playbooks, etc.
+        version: v1.1.0
       - name: cyberark/ansible-conjur-host-identity
         url: https://github.com/cyberark/ansible-conjur-host-identity
         tool: Ansible
         description: Ansible role to provide Conjur machine identity to application
           hosts and install the Summon tool, which enables hosts to securely retrieve
           credentials.
-        version: v0.3.1
+        version: v0.3.2
```
Comment (alexkalish) [inline suite.yml:83]: The first item under "Deprecated" in this release's CHANGELOG entry needs to be moved to "Removed".

**Review by alexkalish — CHANGES_REQUESTED** [review]
Please just change the Ansible host identity CHANGELOG entry.  Otherwise, LGTM.

File: suite.yml (hunk 1 above)
Comment (izgeri) [reply suite.yml:19]: ah, you're right :/

File: suite.yml (hunk 2 above)
Comment (izgeri) [reply suite.yml:83]: @alexkalish can you please approve: https://github.com/cyberark/ansible-conjur-host-identity/pull/47

**Review by alexkalish — APPROVED** [review]

**Review by boazmichaely — APPROVED** [review]
@izgeri **your What's New is amazing !🥇** 
I love the easy going and exciting tone, and the balance of technical vs benefit oriented content.
I've not reviewed the components delta's and the individual changelogs text . No b/w for that, I hope others have (@alexkalish).
Thank you!

**Pull request merged**